_schemaV2.graphql

```diff
@@ -12667,6 +12667,7 @@ type MarketingCollectionQuery {
 enum MarketingCollectionsSorts {
   CREATED_AT_ASC
   CREATED_AT_DESC
+  CUSTOM
```

Comment:
**suggestion:** I think we'd want a more descriptive name for this. The other PR had `EXPLORE_BY`. I could also see other options (such as `CURATED`) which feel maybe less tied to the current UI but still express the fact that this is Artsy's manual ordering.

Reply:
If the `CUSTOM` is not that descriptive let's go with CURATED then.

Reply:
Yeah, I agree that `CUSTOM` is a bit too vague but I like `CURATED` 👍🏼